target/linux/ath79/dts/qca9563_zte_mf286.dtsi

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2021 Cezary Jackiewicz
+// Copyright (c) 2021, 2022 Lech Perczak
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	aliases {
+		led-boot = &led_debug;
+		led-failsafe = &led_debug;
+		led-running = &led_debug;
+		led-upgrade = &led_debug;
+		label-mac-device = &eth0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		/* Hidden green SMD LED below signal strength LEDs
```

Comment:
Hmm, I would move the LED definition completely to the different variants... even if just one has a different color.

(You don't need to do that. But at least remove the "green" here in this dtsi as from what I read the A variant has a blue one?)

Reply:
Makes sense - let's remove "green" from comment, as the node below is referenced in "aliases" node common to all variants, to avoid a dangling reference.